eqllib/analytics/execution/T1170-mshta-ms-office-descendant.toml

```diff
@@ -0,0 +1,21 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies the execution of ``mshta.exe`` as a descendant of a Microsoft Office process."
+id = "d49fc9fe-df80-416d-a861-0be02bef0df5"
+name = "Mshta Descendant of Microsoft Office"
```

Comment:
This is a good one, and I think we can open this up to a handful of other commands or lolbins.

Reply:
I agree, maybe in a future release once we support macros in eqllib?